### PR TITLE
[trivial] typo in README

### PR DESCRIPTION
**Pull request by jnewbery**
Fix a trivial typo in README

### CODE REVIEW

**Comment by carryforward** [comment]
this left me scratching my head too.

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Don't try to port it to mainnet or an altcoin and use it today!  No really.  Lig
   * **Go 1.5 or 1.6**
       
     Installation instructions can be found here: http://golang.org/doc/install. It is recommended to add `$GOPATH/bin` to your `PATH` at this point.
-    **Note:** If you are building with Go 1.5, then you'll need to enable the vendor experiment by setting the `GO15VENDOREXPERIMENT` environment variable to `1`. If you're using Go 1.5, then it is safe to skip this step. 
+    **Note:** If you are building with Go 1.5, then you'll need to enable the vendor experiment by setting the `GO15VENDOREXPERIMENT` environment variable to `1`. If you're using Go 1.6, then it is safe to skip this step.
```
Comment (Roasbeef) [inline README.md]: Minor nit, this should read: 
> If you're using Go 1.6 *or later*...

File: README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ Don't try to port it to mainnet or an altcoin and use it today!  No really.  Lig
   * **Go 1.5 or 1.6**
```
Comment (Roasbeef) [inline README.md]: While you're at it, can you modify this header to read `Go 1.5+`? Thanks!

**Comment by jnewbery** [comment]
nit fixed

**Comment by Roasbeef** [comment]
Thanks! 

LGTM ✨

**Pull request merged**